### ISSUE TITLE
cards with missing properties break IF functions

### ISSUE THREAD

**Issue by brine** [octgn/OCTGN#245]
This bug has driven me crazy for a long time now... it's likely a flaw in the python language but maybe OCTGN can do something about it...

If I have a python function that does something different depending on a card's property (say, Subtype), I would typically do something like:

```
if card.Subtype == "Aura":
```

However, when you pass a card that is MISSING the Subtype (the set def doesn't have the Subtype property for that card), I get TypeError: Value cannot be null.  So in the past OCTGN, I would get around this by writing:

```
if not card.Subtype == None and card.Subtype == "Aura":
```

So that appeared to work fine older OCTGN versions, but lately I've been getting harrassed by the same "value cannot be null" python errors for this.

So basically, if you have cards that may not have a particular property, then you won't be able to make reliable comparisons of that property in python functions.
